modules/virt-enabling-disabling-vm-delete-protection-cli.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-cli_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the command line
+
+To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the command line. You can also disable delete protection for a VM.
+
+By default, delete protection is not enabled for VMs. You must set the option for each individual VM.
+
+.Prerequisites
+
+* You have installed the {oc-first}.
```

Comment:
You also need to know the VMs you want to protect or allow to be deleted, right?

Reply:
Andrea had me remove that prereq in her peer review.